dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskGroupServiceImpl.java

```diff
@@ -253,16 +252,8 @@ public Map<String, Object> queryTaskGroupByStatus(User loginUser, int pageNo, in
     public Map<String, Object> queryTaskGroupByProjectCode(User loginUser, int pageNo, int pageSize, Long projectCode) {
         Map<String, Object> result = new HashMap<>();
         Page<TaskGroup> page = new Page<>(pageNo, pageSize);
-        PageInfo<TaskGroup> emptyPageInfo = new PageInfo<>(pageNo, pageSize);
-        Set<Integer> ids = resourcePermissionCheckService.userOwnedResourceIdsAcquisition(AuthorizationType.TASK_GROUP,
-                loginUser.getId(), log);
```

Comment:
not sure whether this change is suitable or not, I found all resource is users-base, could you please task a look? @WangJPLeo @caishunfeng

Reply:
Thanks for you comment! For me, users can only use task groups created by themselves seems weird, let's see others suggestion :D

Reply:
From my personal perspective, User level permission control is better.

Reply:
In another word, only when the user is granted permission of one project, he can find task groups belongs to the project. Can it be seen as a user-level permission control?

Reply:
> In another word, only when the user is granted permission of one project, he can find task groups belongs to the project. Can it be seen as a user-level permission control?

That's a good question...

Reply:
> All task groups belongs to current project are visible to the user who is authorized project permission.

@Radeity I agree with this.
But it seems that project permission has not been checked in each CURD operation of task group

Reply:
Hi @rickchengx , thanks for reminding, the check should be added, I'll handle this later today :D